recipes/sentry-crashpad/all/conanfile.py

```diff
@@ -41,7 +41,7 @@ def _source_subfolder(self):
     def _minimum_compilers_version(self):
         return {
             "Visual Studio": "15" if tools.Version(self.version) < "0.4.16" else "16",
-            "gcc": "9",
+            "gcc": "5",
```

Comment:
GCC 5 produce strange errors. Lets see what 6 has to say
```suggestion
            "gcc": "6",
```